app/src/main/kotlin/com/hippo/ehviewer/ui/reader/ReaderScreen.kt

```diff
@@ -68,25 +72,38 @@ fun ReaderScreen(info: BaseGalleryInfo, page: Int = -1) {
         }
     }
     val showSeekbar by Settings.showReaderSeekbar.collectAsState()
-    val lazyListState = rememberLazyListState()
-    val zoomableState = rememberZoomableState(zoomSpec = ZoomSpec(maxZoomFactor = 3f))
-    val sync = remember { SliderPagerDoubleSync(lazyListState) }
-    sync.Sync()
-    val isRtl = LocalLayoutDirection.current == Rtl
+    val readingMode by Settings.readingMode.collectAsState { ReadingModeType.fromPreference(it) }
     Deferred({ pageLoader.awaitReady() }) {
+        val lazyListState = rememberLazyListState()
+        val pagerState = rememberPagerState { pageLoader.size }
+        val zoomableState = rememberZoomableState(zoomSpec = ZoomSpec(maxZoomFactor = 3f))
+        val sync = remember { SliderPagerDoubleSync(lazyListState, pagerState) }
+        sync.Sync()
         Box {
+            var appbarVisible by remember { mutableStateOf(false) }
             ReaderAppBars(
-                visible = true,
-                isRtl = isRtl,
+                visible = appbarVisible,
+                isRtl = readingMode == ReadingModeType.RIGHT_TO_LEFT,
                 showSeekBar = showSeekbar,
                 currentPage = sync.sliderValue,
                 totalPages = pageLoader.size,
                 onSliderValueChange = { sync.sliderScrollTo(it + 1) },
                 onClickSettings = { },
                 modifier = Modifier.zIndex(1f),
             )
-            LazyColumn(modifier = Modifier.fillMaxSize().zoomable(zoomableState), state = lazyListState) {
-                items(pageLoader.mPages) { page ->
+            CompositionLocalProvider(LocalTextStyle provides MaterialTheme.typography.bodySmall) {
+                PageIndicatorText(
+                    currentPage = sync.sliderValue,
+                    totalPages = pageLoader.size,
+                    modifier = Modifier.align(Alignment.BottomCenter).zIndex(0.5f),
```

Comment:
Navigation bar padding, same for ReaderAppBars.

Reply:
Done, cannot do it for ReaderAppBars now, must control systemBar visibility & color first